src/main/java/net/geforcemods/securitycraft/network/ClientProxy.java

```diff
@@ -237,6 +237,19 @@ public static void onFMLClientSetup(FMLClientSetupEvent event)
 		KeyBindings.init();
 	}
 
+	private int mixTints(int tint1, int tint2)
+	{
+		int red = (tint1 >> 0x10) & 0xFF;
+		int green = (tint1 >> 0x8) & 0xFF;
+		int blue = tint1 & 0xFF;
+
+		red *= (float)(tint2 >> 0x10 & 0xFF) / 0xFF;
+		green *= (float)(tint2 >> 0x8 & 0xFF) / 0xFF;
+		blue *= (float)(tint2 & 0xFF) / 0xFF;
+
+		return ((red << 8) + green << 8) + blue;
+	}
+
```

Comment:
Move this below the tint method, please. It doesn't really matter for functionality, but it makes more logical sense to me.